### PR TITLE
Changes on nuttx getting started article

### PR DESCRIPTION
**Pull request by eren-terzioglu**
## Description

Change on nuttx getting started article. Suggestion came from @georgik

### CODE REVIEW

**Comment by eren-terzioglu** [comment]
@tmedicci, @fdcavalcanti I am open to your suggestions.

**Comment by georgik** [comment]
LGTM. Waiting for review from @tmedicci  or @fdcavalcanti

**Comment by fdcavalcanti** [comment]
lgtm as well.

**Review by tmedicci — APPROVED** [review]

**Review by pedrominatel — APPROVED** [review]

File: content/blog/nuttx-getting-started/index.md
Comment (f-hollow) [inline content/blog/nuttx-getting-started/index.md:196]: Please add a comma. It helps to visually separate the first part of the sentence saying what will happen from the second part of the sentence saying what you need to do to make it happen.

```suggestion
NuttX provides ready-to-use board default configurations that enable the required config (from Kconfig) for a use scenario, such as Wi-Fi or I2C. To enter NuttX directory and list all available configurations for the ESP32 DevKitC V4 board, you can use the following command:
```

If I may, I would like to propose further improvements to the sentence. The command `cd` is self-evident to most people, so the part `To enter NuttX directory` sounds like partonizing. How about this:

To list all available configurations for the ESP32 DevKitC V4 board, go to your `nuttx` directory and run:

**Comment by f-hollow** [comment]
@eren-terzioglu Please check my comment, and then let's merge this PR.

File: content/blog/nuttx-getting-started/index.md
Comment (eren-terzioglu) [reply content/blog/nuttx-getting-started/index.md:196]: How about this:

"To list all available configurations for the ESP32 DevKitC V4 board, go to your `nuttxspace` directory and run:"

Directory structure looks like this:

```
.
└── nuttxspace/
    ├── nuttx
    └── apps
```

If we say go to your nuttx directory, we don't need to add `cd nuttx` command, I think. This is main purpose of this PR.

**Review by f-hollow — APPROVED** [review]

**Pull request merged**
